docs/internal/config/Element-Type-Specs.md

```diff
@@ -72,6 +72,7 @@ _Free-form, single-line text input element_
 - **description**: `string` -- additional explanatory text (usually not required) [Optional]
 - **placeholder**: `string`-- text to display before user input (HTML "placeholder" attribute) [Optional]
 - **default**: `string` -- default response, will change dynamically in response to form changes until the user has edited it [Optional]
+- **replaceResponseOnDefaultChange**: `boolean` -- if `true`, when the `default` value changes, the current response will be replaced by the new default. Otherwise (`false`), the default will only be used if there is no response yet. [Optional; default `true`]
```

Comment:
There _are_ occasional cases where we *don't* want a value to change after a user has already edited it. In those occasional times, we can use this new parameter. But for the most part, we can ignore it, as it won't apply very often.

Reply:
I think it would be clearer if you change this property name and use the opposite default value (and swap the logic), just because some property as `persistUserInput` seems a bit clearer than this long property.
So you `persistUserInput` default to `false` we should update the element in case `default` VALUE changes

Reply:
Another comment: I not add this property explanation under the common properties (line 30) ?

Reply:
> I think it would be clearer if you change this property name and use the opposite default value (and swap the logic), just because some property as `persistUserInput` seems a bit clearer than this long property.
> So you `persistUserInput` default to `false` we should update the element in case `default` VALUE changes

Yeah, okay good idea. Will do.

> Another comment: I not add this property explanation under the common properties (line 30)

I did think of that, but the "common properties" are ones that have their own database field in the `template_element` table, whereas this one is still a JSON parameter (even though it's available to all ). I did consider making it a "common" field, but I thought that would be a bit unnecessary and involve extra migration etc.. Also, doing that might add to the confusion between the dynamic default parameter and the common "defaultValue" parameter.

And since "default" is a JSON parameter for each plugin, it makes sense to leave another parameter relating to it in the same place.

I do agree though that it's confusing to have a global "defaultValue" field as well as per-element "defaults".